src/Magnum/DartIntegration/Test/DartSkeletonTest.cpp

```diff
@@ -0,0 +1,267 @@
+#include <dart/dynamics/BallJoint.hpp>
+#include <dart/dynamics/BodyNode.hpp>
+#include <dart/dynamics/BoxShape.hpp>
+#include <dart/dynamics/CylinderShape.hpp>
+#include <dart/dynamics/EllipsoidShape.hpp>
+#include <dart/dynamics/RevoluteJoint.hpp>
+#include <dart/dynamics/Skeleton.hpp>
+#include <dart/simulation/World.hpp>
+#include <dart/utils/urdf/urdf.hpp>
+
+#include <Corrade/TestSuite/Tester.h>
+
+#include <Magnum/SceneGraph/MatrixTransformation3D.h>
+#include <Magnum/SceneGraph/Object.hpp>
+#include <Magnum/SceneGraph/SceneGraph.h>
+#include <Magnum/Trade/ImageData.h>
+#include <Magnum/Trade/MeshData3D.h>
+#include <Magnum/Trade/PhongMaterialData.h>
+#include <Magnum/Trade/TextureData.h>
+
+#include <Magnum/DartIntegration/ConvertShapeNode.h>
+#include <Magnum/DartIntegration/DartSkeleton.h>
+
+#include <Magnum/DartIntegration/Test/configure.h>
+
+const double default_height = 1.0; // m
+const double default_width = 0.2; // m
+const double default_depth = 0.2; // m
+
+const double default_torque = 15.0; // N-m
+const double default_force = 15.0; // N
+const int default_countdown = 200; // Number of timesteps for applying force
+
+const double default_rest_position = 0.0;
+const double delta_rest_position = 10.0 * M_PI / 180.0;
```

Comment:
Wanted to suggest using `constexpr Rad delta_rest_position = 10.0_degf` here, but the variable isn't used anywhere, if I see correctly.